### PR TITLE
Install a stable version of parallel

### LINKED ISSUE #2524: Travis error because of parallel
```
gpg: requesting key FFFFFFF1 from hkp server keys.gnupg.net
gpg: key FFFFFFF1: public key "Ole Tange <ole@tange.dk>" imported
gpg: key FFFFFFF1: public key "Ole Tange <ole@tange.dk>" imported
gpg: no ultimately trusted keys found
gpg: Total number processed: 2
gpg:               imported: 2  (RSA: 1)
gpg: requesting key 88888888 from hkp server keys.gnupg.net
gpgkeys: key 88888888 not found on keyserver
gpg: no valid OpenPGP data found.
gpg: Total number processed: 0
The signature on parallel-20170522.tar.bz2 is wrong. This may indicate that a criminal has changed the code.
THIS IS BAD AND THE CODE WILL NOT BE INSTALLED.
See http://git.savannah.gnu.org/cgit/parallel.git/tree/README for other installation methods.
The command "(wget -O - pi.dk/3 || curl pi.dk/3/ || fetch -o - http://pi.dk/3) | bash" failed and exited with 1 during .

```

### PR DESCRIPTION
**Pull request by NeelMohapatra**
Fixes #2524

### CODE REVIEW

**Comment by NeelMohapatra** [comment]
Someone needs to file a bug with apt-get and parallel to update their lists.

**Comment by domoritz** [comment]
Can you explain why these commands are the way they are? Why is chmod necessary? How does this resolve the problems we saw?

**Comment by NeelMohapatra** [comment]
I'm installing it by getting the files directly from their site. That prevents the need to match checksums. I think the chmod is necessary for the command to be executable. Then I'm just moving the program to \bin

**Comment by domoritz** [comment]
Please just use `chmod +x` then. And to make the program findable, please put it in `/usr/local/bin` or change the `PATH`.

**Comment by NeelMohapatra** [comment]
Why are these changes necessary?

**Comment by domoritz** [comment]
Because I want to understand what's happening and we should make sure this is a clean install so that it doesn't keep breaking.

File: .travis.yml [hunk 1]
```diff
@@ -4,10 +4,11 @@ node_js:
   - "7"
   - "8"
 before_install:
-  - wget http://git.savannah.gnu.org/cgit/parallel.git/plain/src/parallel # Minimal installation
-  - chmod 755 parallel
+  - # Minimal installation as desdcribed in https://github.com/martinda/gnu-parallel
+  - wget http://git.savannah.gnu.org/cgit/parallel.git/plain/src/parallel
+  - chmod 700 parallel # Need rwx permissoins for user.
```
Comment (domoritz) [inline .travis.yml]: Not sure why you need write and read permissions. `chmod + x parallel` may be sufficient.

Comment (NeelMohapatra) [inline .travis.yml]: Okay. I just put read/write permissions in case the system needed to alter files to execute use parallel.

Comment (domoritz) [inline .travis.yml]: Read and write is for the file, not the executable.

File: scripts/generate-images.sh [hunk 2]
```diff
@@ -8,4 +8,4 @@ set -e
 export NODE_PATH=.
 
 echo "Generating SVGs..."
-ls examples/specs/*.vl.json | parallel --eta --halt 1 "bin/vl2svg {} build/examples/images/{/.}.svg"
+ls examples/specs/*.vl.json |  --no-notice --eta --halt 1 "bin/vl2svg {} build/examples/images/{/.}.svg"
```
Comment (NeelMohapatra) [inline scripts/generate-images.sh]: @kanitw @domoritz 😂

Comment (NeelMohapatra) [inline scripts/generate-images.sh]: I can't believe we missed this.

Comment (domoritz) [inline scripts/generate-images.sh]: Ups.

**Pull request merged**